src/sass/components/_articles.scss

```diff
@@ -1,8 +1,15 @@
 .articles {
   padding: 56px 0 40px 0;
-  margin: 20px 0 0;
+  margin: 20px auto 0;
```

Comment:
just a note, you use top margin here. you can consider refactoring this code to use bottom margin as everywhere else